### ISSUE TITLE
ConvertedResultIterator does not handle custom types in schemas.

### ISSUE THREAD

**Issue by chanmix51** [pomm-project/Foundation#53]
This is mainly due to [the use](a6d5e15fc2d21153375c6c983d8b5bfe40916c0a) of `pg_get_type` function in the `ResultHandler` wrapper class. 

Apparently, this function does only output the name of the type regardless of its schema. This makes Pomm to look for a converter declared upon a type without schema prefix which can lead to collisions.

**Comment by stof** [comment]
the link is broken

**Comment by chanmix51** [comment]
True, thank you for reporting :+1:  Here is [the line](https://github.com/pomm-project/Foundation/blob/2.0/sources/lib/Session/ResultHandler.php#L156).

**Comment by chanmix51** [comment]
What we can do is to report the problem to PHP’s `pg_` lib maintainer.
